js/_website/generate_jsons/src/guides/__init__.py

```diff
@@ -0,0 +1,164 @@
+import os
+import markdown2
+import shutil
+import re
+import json
+
+# DIR = os.path.dirname(__file__)
+
+GRADIO_DIR = "../../../"
+GUIDES_DIR = os.path.join(GRADIO_DIR, "guides")
+GUIDE_ASSETS_DIR = os.path.join(GUIDES_DIR, "assets")
+DEMOS_DIR = os.path.join(GRADIO_DIR, "demo")
```

Comment:
Maybe we could just define the base path or root path somewhere once and use it in all of these modules?